src/Identity.API/README.md

```diff
@@ -1,14 +0,0 @@
-# Containerized eShop - Identity Service
```

Comment:
deleted these readmes as they seemed legacy/low value. we can add new ones for each project later if we want.